test/Sentry.Profiling.Tests/SamplingTransactionProfilerTests.cs

```diff
@@ -319,6 +319,13 @@ async Task VerifyAsync(HttpRequestMessage message)
             options.TransactionProfilerFactory.Should().BeNull();
             using var hub = (SentrySdk.InitHub(options) as Hub)!;
             SampleProfilerSession.ThrowOnNextStartupForTests.Should().BeFalse();
+
+            if (options.TransactionProfilerFactory is SamplingTransactionProfilerFactory factory)
+            {
+                factory.StartupTimedOut.Should().BeTrue();
+                Skip.If(TestEnvironment.IsGitHubActions, "Session sometimes takes too long to start in CI.");
```

Comment:
I guess this will still fail in our local dev environments. If that's the case though (and it stops failing in CI) at least we know what's causing the issue.

Reply:
yes, that was the idea